tests/BootstrapperTest.php

```diff
@@ -82,12 +85,16 @@
     expect(DB::table('users')->first()->name)->toBe('Foo');
 });
 
-test('cache data is separated', function () {
+test('cache data is separated', function (string $bootstrapper) {
+    $cacheDriver = 'redis';
+
+    if ($bootstrapper === PrefixCacheTenancyBootstrapper::class) {
+        PrefixCacheTenancyBootstrapper::$tenantCacheStores = [$cacheDriver];
```

Comment:
Could we move this to `beforeEach()` and get rid of the `if` completely? And then we could get rid of the `$cacheDriver` variable as well.

We'd undo the change to `tenantCacheStores` in `afterEach()`.

Not 100% sure about this change, but it'd clean up this test so it would actually just accept the `$bootstrapper` and work with it without having any logic coupled to the values that are actually passed.

Reply:
Yeah, we can do that. The only thing that'd be worse would be that there won't be the `$cacheDriver` variable, so in the test, we'd have to assume 'redis' is in the tenant cache stores property

Reply:
Are we using the cache driver variable anywhere in the actual test logic? As in after configuring these classes/the config,

Reply:
Yeah, after this line, we use the variable to set the config ([`'cache.default'`](https://github.com/archtechx/tenancy/pull/1014/files/3481e24c7d36dbdcc5e010a995f367ba38a86c04#diff-b22931f4c76926f01513d299f2f87fb1528e0d3a92e539ef66cb4675b4342f4bR97))

Reply:
Well that's still in what I meant by the config. My point is that if we move the `PrefixCacheTenancyBootstrapper::$tenantCacheStores` logic elsewhere, we don't need the variable anymore probably.

> so in the test, we'd have to assume 'redis' is in the tenant cache stores property

We'd probably set this in the `beforeEach()`. Tests can make assumptions about the environment like that.

Reply:
Do you mean doing it like this? https://github.com/archtechx/tenancy/pull/1014/commits/13b85a2d3be9041d069f00f6e7306d4a9ad32290

Reply:
Yeah. Though if `cache.default` is never set to anything except `redis`, there's no need to make this assertion at the start of the file:
```php
expect(config('cache.default'))->toBe('redis');
```

since we can assume that it will have the value defined in `beforeEach()`. The line doesn't hurt, but doesn't seem to add anything either right now, so we can probably remove it.